### ISSUE TITLE
LDAP integration

### ISSUE THREAD

**Issue by Marc176** [ThomasLeister/webmum#38]
Hi,

would it be possible to get a ldap integration/import? Maybe just pull a whole list from LDAP and create accounts?

**Comment by ohartl** [comment]
Yes, it is possible since php has easy ldap functionality. Currently I'm not positive on implementing this feature on myself soon, since it's not very interesting for me right now. But you or others are free to make a detailed list of requirements, develop this idea and make pull requests. I would be happy to review your code.
